<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(164, 208, 255);</string>
  </property>
  <widget class="QPlainTextEdit" name="quote_le">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>641</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="comment_le">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>641</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Цитата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Комментарий:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(216, 248, 255);</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
